src/UXClient/Models/ChartOptions.ts

```diff
@@ -13,7 +16,9 @@ class ChartOptions {
     public fromChart: boolean; // whether a component is a subcomponent of another one or is a standalone
     public grid: boolean; // whether the chart includes a grid and grid button
     public hideChartControlPanel: boolean; // whether to hide the panel with chart control buttons
+    public includeEnvelope: boolean; //whether to include an area showing min/max boundaries in the line chart
```

Comment:
please add another example in index.html showing envelope and step interpolation